src/core/meshcop/joiner.hpp

```diff
@@ -109,12 +109,29 @@ class Joiner : public InstanceLocator
      */
     void GetJoinerId(Mac::ExtAddress &aJoinerId) const;
 
+    /**
+     * This method validates the PSKd.
+     *
+     * Per Thread specification, a Joining Device Credential is encoded as
+     * uppercase alphanumeric characters (base32-thread: 0-9, A-Z excluding
+     * I, O, Q, and Z for readability) with a minimum length of 6 such
+     * characters and a maximum length of 32 such characters.
+     *
+     * param[in]  aPskd  The PSKd to validate.
```

Comment:
```suggestion
     * param[in]  aPskd  The PSKd to validate.
     *
```
Add blank line between params and retvals